src/Authentication/Authentication/Cmdlets/ConnectMgGraph.cs

```diff
@@ -180,12 +188,14 @@ private async Task ProcessRecordAsync()
                             authContext.TokenCredentialType = TokenCredentialType.ClientCertificate;
                         }
                         break;
-                    case Constants.AccessTokenParameterSet:
+                    case Constants.AppSecretCredentialParameterSet:
                         {
-                            authContext.AuthType = AuthenticationType.UserProvidedAccessToken;
-                            authContext.TokenCredentialType = TokenCredentialType.UserProvidedAccessToken;
-                            authContext.ContextScope = ContextScope.Process;
-                            GraphSession.Instance.InMemoryTokenCache = new InMemoryTokenCache(Encoding.UTF8.GetBytes(new NetworkCredential(string.Empty, AccessToken).Password));
+                            authContext.AuthType = AuthenticationType.AppOnly;
```

Comment:
Won't this cause user confusion when users inevitably try to use `-Credential` within `Connect-MgGraph` for ROPC / User Credential auth flow use with delegated permission types? I get that you're only intending to add support for client secrets / App Only, I just think in the process it might be creating UX confusion. 

Perhaps an alias or rename of the parameter could aleviate this. `-ClientSecret`, `-ClientSecretCredential`, `-AppCredential`, or `-AppSecret` might be a few possibilities.